src/app/components/Header/SimplifiedHeader.tsx

```diff
@@ -6,8 +6,8 @@ import { TestingInfo } from "../TestingInfo/TestingInfo";
 export const SimplifiedHeader = () => {
   return (
     <nav>
-      <div className="bg-base-300 shadow-sm">
-        <div className="container mx-auto flex w-full items-center justify-between gap-4 p-6 pb-4 md:pb-6">
+      <section className="bg-primary-main dark:bg-[#181818] h-[300px] mb-[-188px]">
+        <div className="container h-[84px] py-[22px] px-6 mx-auto flex items-center justify-between">
```

Comment:
lets better use `h-20` and `py-6` instead. they are not exact equal to this values but its better to avoid "custom" values